contrib/babelfishpg_tsql/src/dbcmds.c

```diff
@@ -1225,3 +1258,125 @@ create_guest_schema_for_all_dbs(PG_FUNCTION_ARGS)
 
 	PG_RETURN_INT32(0);
 }
+
+
+/*
+* This function is only being used during upgrade to v4.4.0
+* to create database roles db_accessadmin for each database
+*/
+PG_FUNCTION_INFO_V1(create_database_roles_for_all_dbs);
+Datum
+create_database_roles_for_all_dbs(PG_FUNCTION_ARGS)
```

Comment:
Why not make the name specific to what role it is creating or do we intend to add more roles with this function in this release?

Reply:
Yes, three more fixed db roles will use this.

Reply:
Can we move the creation of db roles to another function like `create_db_roles_if_not_exists` mentioned here? https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/3018/files 

Since our PRs have common changes, that would be better to add other db roles in the same function.